### PR TITLE
Validation checks for organisation type form

### PR DESCRIPTION
**Pull request by stuartmccoll**
Raised issue #40 to revisit this as a piece of technical debt at a later date.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -22,7 +22,6 @@
           <%= yield %>
         </div>
       </div>
-    </div>
```
Comment (gidsg) [inline app/views/layouts/application.html.erb]: is this change intentional?

File: app/controllers/organisation/organisation_type_controller.rb [hunk 2]
```diff
@@ -2,49 +2,47 @@ class Organisation::OrganisationTypeController < ApplicationController
...
 
-        flash[:alert] = @@error_message
+    else
 
-      else
+      logger.debug "Organisation type not found when attempting to update organisation ID: " +
+                       "#{@organisation.id}"
 
-        logger.debug "Finished updating organisation ID: #{@organisation.id}"
+      flash[:alert] = @organisation.errors[:org_type][0]
```
Comment (gidsg) [inline app/controllers/organisation/organisation_type_controller.rb]: I think we could make an error handler that iterates through this hash, displaying the errors at the correct location, which we could re-use as a partial, rather than writing bespoke code for each page. But happy to do this for now and we can re-visit potentially.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (stuartmccoll) [reply app/views/layouts/application.html.erb]: I'm not sure how this branch ended up with an extra closing `div` tag, but this line is taking it out - looking at the file in full, the number of opening and closing tags matches what's in `master` now.

File: app/views/organisation/organisation_type/type.html.erb [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+      </ul>
+    </div>
+  </div>
+
+<% end %>
 
 <%= form_for @organisation, url: organisation_organisation_type_put_url, method: :put do |f|  %>
 
-  <div class="govuk-form-group">
+  <div class="govuk-form-group <% if flash[:alert] %>govuk-form-group--error<% end %>">
```
Comment (gidsg) [inline app/views/organisation/organisation_type/type.html.erb]: We could express this slightly more concisely as:
`<div class="govuk-form-group <% "govuk-form-group--error"  if flash[:alert] %>">`
